### PR TITLE
Adding 2 Negative Test cases

### PR DESCRIPTION
**Pull request by hirentha**

### CODE REVIEW

File: common/lib/port_config.py [hunk 1]
```diff
@@ -68,6 +68,21 @@ def gnmi_ctl_set(self, params):
...
+            :type params: str
+            :return: stdout of the gnmi-cli set command
+            :rtype: str
+            """
+            cmd = self.form_cmd(f'set "{params}"')
+            output, return_code, _ = self.local.execute_command(cmd)
+            if "Set request, successful" not in output:
+                return True
+            else:
+                raise ExecuteCMDException(f"Max+1 port created")
```
Comment (dkaranjai) [inline common/lib/port_config.py]: remove 'f'

File: common/utils/gnmi_ctl_utils.py [hunk 2]
```diff
@@ -41,6 +41,20 @@ def gnmi_set_params(params):
     return output
 
 
+def gnmi_set_params_max_plus1_ports(params):
+    port_config = PortConfig()
```
Comment (dkaranjai) [inline common/utils/gnmi_ctl_utils.py:53]: add function description

File: tests/nt_out_range_port.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        test_params = test_params_get()
+        config_json = test_params["config_json"]
+        self.config_data = get_config_dict(config_json)
+
+        self.gnmictl_params = get_gnmi_params_simple(self.config_data)
+
+    def runTest(self):
+        socket_path_dir = self.config_data["socket_path_dir"]
+        max_port_count = int(self.config_data["max_port_count"])
+        check_vhost = check_and_clear_vhost(socket_path_dir)
```
Comment (dkaranjai) [inline tests/nt_out_range_port.py:56]: add comment

File: tests/nt_send_traffic_non_existing_group.py [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+            udp_sport=self.config_data["traffic"]["in_pkt_header"]["udp_port"][0],
+            udp_dport=self.config_data["traffic"]["in_pkt_header"]["udp_port"][1],
+        )
+        send_packet(self, port_ids[self.config_data["traffic"]["send_port"][3]], pkt)
+        try:
+            verify_no_packet(
+                self, pkt, port_ids[self.config_data["traffic"]["receive_port"][1]][1]
+            )
+            log.passed(
+                f" Verification of udp packet check passed : No traffic from TAP3 to TAP1 port"
```
Comment (dkaranjai) [inline tests/nt_send_traffic_non_existing_group.py]: remove 'f'

File: tests/nt_send_traffic_non_existing_group.py [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+            udp_sport=self.config_data["traffic"]["in_pkt_header"]["udp_port"][0],
+            udp_dport=self.config_data["traffic"]["in_pkt_header"]["udp_port"][1],
+        )
+        send_packet(self, port_ids[self.config_data["traffic"]["send_port"][3]], pkt)
+        try:
+            verify_no_packet(
+                self, pkt, port_ids[self.config_data["traffic"]["receive_port"][2]][1]
+            )
+            log.passed(
+                f" Verification of udp packet check passed : No traffic from TAP3 to TAP2 port"
```
Comment (dkaranjai) [inline tests/nt_send_traffic_non_existing_group.py]: removed 'f'

**Review by dkaranjai — CHANGES_REQUESTED** [review]

**Review by sjwahub — APPROVED** [review]

**Comment by sjwahub** [comment]
Unit test log?

**Comment by hirentha** [comment]
[Negative_Test_Cases_Logs.txt](https://github.com/ipdk-io/ptf-tests/files/10769266/Negative_Test_Cases_Logs.txt)

**Review by sjwahub — APPROVED** [review]

**Review by dkaranjai — APPROVED** [review]

**Pull request merged**